app/jjit.py

```diff
@@ -0,0 +1,135 @@
+from bs4 import BeautifulSoup
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+
+import pandas as pd
+import time
+import re
+import os
+
+
+def scrap_data_jjit(driver, offers, links):
+    soup = BeautifulSoup(driver.page_source, 'html.parser')
+
+    for offer in soup.find_all('div', class_="css-2crog7"):
+
+        link = offer.find_all('a', class_="offer_list_offer_link css-4lqp8g")[0]['href']
+
+        if link in links:
+            continue
+
+        links.append(link)
+
+        name = offer.find_all('h2', class_="css-1gehlh0")[0].text
+        company = offer.find_all('div', class_="css-aryx9u")[0].text
+        salary = offer.find_all('div', class_="css-17pspck")[0].text
+        location = offer.find_all('div', class_="css-11qgze1")[0].text
+        try:
+            work_mode = offer.find_all('div', class_="css-7ktfgf")[0].text
+        except IndexError:
+            work_mode = "Not specified"
+
+        techs = [tech.text for tech in
+                 offer.find_all('div', class_="css-yicj0q")[0].find_all('div', class_='css-1am4i4o')]
+
+        new_offer = [name, company, salary, location, work_mode, techs, link]
+
+        offers.append(new_offer)
+
+    return offers, links
+
+def jjit_scroll(url):
+
+    current_dir = os.path.dirname(os.path.abspath(__file__))
+    driver_path = os.path.join(current_dir, 'chromedriver.exe')
+    service = Service(executable_path=driver_path)
+    driver = webdriver.Chrome(service=service)
```

Comment:
Gdybyś miał ustaloną strukturę projektu, to nie byłoby potrzeby pobierać ścieżki bezwzględnej i potem jej łączyć, tylko wystarczyłoby podać ścieżkę względną. Aczkolwiek ze ścieżkami w Pythonie łatwo się zaplątać, więc być może próbowałeś ale nie działało.

Poza tym, myślę że lepiej sprawdziłoby się `os.getcwd()`